__tests__/app.test.js

```diff
@@ -40,7 +40,47 @@ describe("api/topics", () => {
       .get("/api/topic")
       .expect(404)
       .then((response) => {
-        expect(response.body.msg).toBe("page not found");
+        expect(response.body.msg).toBe("Page not found");
+      });
+  });
+});
+
+describe("/api/topics/articles/:article_id", () => {
+  test("GET:200 - sends a single article to the client", () => {
+    return request(app)
+      .get("/api/articles/3")
+      .expect(200)
+      .then((response) => {
```

Comment:
your response should be a single object rathen than an object nested in an array.